addon/scope.js

```diff
@@ -0,0 +1,60 @@
+class Scope {
```

Comment:
that is true.

Is there a JavaScript/ES6/Ember analog to `method_missing`?

We'd need a **method**\_missing, not a property missing (which Ember and ES6 support).

If we could manage to declare a method missing, the implementation could be consolidated to one method that would support all future test helper extensions